scripts/copy-azure-teacher.py

```
@@ -0,0 +1,414 @@
+KEY_VAULT_NAME = "kv-lt-dev-5ba22u"
+KEY_VAULT_SECRET = "ConnectionStrings--Default"
+
+# Local Docker SQL connection
+LOCAL_SERVER = "tcp:127.0.0.1,1434"
+LOCAL_DATABASE = "LangTeach"
+
+
+def get_azure_connection_string():
+    """Fetch the Azure SQL connection string from Key Vault via az CLI."""
+    # On Windows, az is a .cmd script; invoke via cmd /c so it resolves via PATH
+    result = subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)